worldengine/cli/main.py

```diff
@@ -335,9 +336,24 @@ def main():
                                action="store_true",
                                help="Draw outer land border",
                                default=False)
-
     # TODO: allow for RGB specification as [r g b], ie [0.5 0.5 0.5] for gray
 
+    # -----------------------------------------------------
+    export_options = parser.add_argument_group(
+        "Export Options", "You can specify the formats you wish the generated output to be in. ")
+    export_options.add_argument("--export-type", dest="export_type",
+                                help="Export to a specific format such as: BMP or PNG",
+                                default="bmp")
+    export_options.add_argument("--export-bpp", dest="export_bpp", type=int,
+                                help="Bits per pixel: 8, 16 and 32",
+                                default=8)
+    export_options.add_argument("--export-signed", dest="export_signed", action="store_true",
```

Comment:
It is in the help text, if you don't know what it means... then best to stay away from it. ;)

Besides, if you export to a format that doesn't supported signed ints, then it will yell at you anyway.

CLI isn't for dummies, that is what the GUI is for. The target audience of the CLI will likely know what signed means.